syntax = "proto3";
import "common.proto";

service FileProcess {
  rpc ProcessFile(stream FileChunk) returns (stream ProgressResponse);
}
